### PR TITLE
sous-chef adoption

### PR DESCRIPTION
**Pull request by derekgroh**
# Description

Transfers cookbook ownership to sous-chefs

## Check List

- [X] A summary of changes made is included in the CHANGELOG under `## Unreleased`
- [X] New functionality includes testing.
- [X] New functionality has been documented in the README if applicable.

### CODE REVIEW

File: .overcommit.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    enabled: true
+    required_executable: 'rspec'
+  Cookstyle:
+    enabled: true
+    required_executable: 'cookstyle'
+    command: ["cookstyle"]
+  Delivery:
+    enabled: true
+    required_executable: 'delivery'
+    flags: ['local', 'all']
```
Comment (damacus) [inline .overcommit.yml:17]:
```suggestion
```

File: Berksfile [hunk 2]
```diff
@@ -1,5 +1,8 @@
-cookbook 'windows_print'
+source 'https://supermarket.chef.io'
+
+metadata
 
 group :integration do
-  cookbook "windows_print", :path => "."
+  cookbook 'test_windows_print', path: 'test/fixtures/cookbooks/test_windows_print'
+  cookbook 'test_windows_ad', path: 'test/fixtures/cookbooks/test_windows_ad'
```
[the remark this answers is not in the document]
Comment (derekgroh) [reply Berksfile:7]: This is provided for local testing as LDP feature requires the print server to be a domain member. I haven't seen this be successful in GHAs, so unsure how best to approach supporting both models.

[the remark this answers is not in the document]
Comment (damacus) [reply Berksfile:7]: We can optionally enable AD joining in Test Kitchen maybe?

[the remark this answers is not in the document]
Comment (derekgroh) [reply Berksfile:7]: Perhaps, I've not been able to successfully replicate a case of the GHA node rebooting after ADDS install without an exit 20 code from test kitchen.

[the remark this answers is not in the document]
Comment (damacus) [reply Berksfile:7]: Ah! If it's just a reboot we need to work around this should sort it out

https://github.com/sous-chefs/selinux/blob/main/kitchen.yml#L15-L16

[the remark this answers is not in the document]
Comment (derekgroh) [reply Berksfile:7]: Is there anything else that needs to be added, still getting exit 20

**Pull request merged**